From d0bf28fd92ad1dfe056c62b0cba646f82ae4226e Mon Sep 17 00:00:00 2001
From: Chris Goldsworthy <cgoldswo@codeaurora.org>
Date: Thu, 10 Sep 2020 08:39:38 -0700
Subject: [PATCH 767/830] ANDROID: gki_defconfig: enable
 IOMMU_LIMIT_IOVA_ALIGNMENT
X-NVConfidentiality: public

Enable IOMMU_LIMIT_IOVA_ALIGNMENT with the default setting
of IOMMU_IOVA_ALIGNMENT (2MB alignment)

Bug: 148141615
Signed-off-by: Todd Kjos <tkjos@google.com>
Signed-off-by: Chris Goldsworthy <cgoldswo@codeaurora.org>
Change-Id: I1e92fb14dd0518746e5e53fab80234915a7f0d68
---
 arch/arm64/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index ef49759ca003..550f539aed50 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -426,6 +426,7 @@ CONFIG_COMMON_CLK_SCPI=y
 CONFIG_HWSPINLOCK=y
 # CONFIG_SUN50I_ERRATUM_UNKNOWN1 is not set
 CONFIG_MAILBOX=y
+CONFIG_IOMMU_LIMIT_IOVA_ALIGNMENT=y
 CONFIG_REMOTEPROC=y
 CONFIG_REMOTEPROC_CDEV=y
 CONFIG_RPMSG_CHAR=y
-- 
2.17.1

